internal/codegen/golang/result.go

```diff
@@ -73,10 +73,17 @@ func buildStructs(r *compiler.Result, settings config.CombinedSettings) []Struct
 				Comment: table.Comment,
 			}
 			for _, column := range table.Columns {
+				tags := map[string]string{}
```

Comment:
@kyleconroy This feels a little clunky -- happy to try and do something fancier/less repetitive.

Though given its only in three places (one of which is deprecated I think?), maybe not too bad in current form?

Reply:
I'm all for clunky and repetitive. The generation code is already a bit of a mess and I don't think this makes it any worse.